R/pkg/R/sparkR.R

```diff
@@ -362,6 +362,10 @@ sparkR.session <- function(
   enableHiveSupport = TRUE,
   ...) {
 
+  if (length(sparkConfig[["spark.sql.warehouse.dir"]]) == 0) {
+    sparkConfig[["spark.sql.warehouse.dir"]] <- tempdir()
```

Comment:
@felixcheung I'm confused. By "spark property" do you mean something passed to `sparkR.session()` via the `sparkConfig` argument?